super-mode-calculator/src/lambdas/calculate/lambda.ts

```diff
@@ -0,0 +1,30 @@
+import * as AWS from 'aws-sdk';
+import type { QueryExecutionId } from 'aws-sdk/clients/athena';
+import { getCheckedExecutionResult } from '../../lib/query';
+import { queryActiveArticles, writeRows } from './dynamo';
+import { parseResult } from './parse';
+import { isCurrentlyInSuperMode, shouldEnterSuperMode } from './superMode';
+
+const STAGE: string = process.env.Stage ?? 'PROD';
+
+const athena = new AWS.Athena({ region: 'eu-west-1' });
+const docClient = new AWS.DynamoDB.DocumentClient({ region: 'eu-west-1' });
+
+export async function handler(executionIds: QueryExecutionId[]): Promise<void> {
+	const executionId = executionIds[0];
+	const result = await getCheckedExecutionResult(executionId, athena);
+	const rows = parseResult(result);
+
+	console.log({ rows });
```

Comment:
Do we still need the console logging?

Reply:
We were thinking that this might be useful for debugging purposes. It just runs once an hour so the logs shouldn't be too noisy